src/ClassCache.php

```diff
@@ -82,6 +83,10 @@ public function getClassPath(string $className, string $baseProxyClassName): str
     {
         [$classFileName, $classFilePath] = $this->getClassFileNameAndPath($className, $baseProxyClassName);
 
+        if ($this->cachePath === 'php://memory') {
```

Comment:
Just noticed that every time memory stream is used, a bunch of proxy files is left after running tests (location is repo root folder). :thinking: